packages/utilities/src/dom/getTopMostDocument.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Helper to get the top-most document object.
+ *
+ * @public
+ */
+export function getTopMostDocument(): Document | undefined {
+  let current = window;
```

Comment:
Should [getDocument](https://github.com/OfficeDev/office-ui-fabric-react/blob/a97cc74c3b78c50bb244bb880dfeb8fbc238f283/packages/utilities/src/dom/getDocument.ts#L10) or [getWindow](https://github.com/OfficeDev/office-ui-fabric-react/blob/a97cc74c3b78c50bb244bb880dfeb8fbc238f283/packages/utilities/src/dom/getWindow.ts#L22) be used here?

Reply:
I wasn't aware of `getWindow`, but it would make sense to keep it consistent, even though there's no difference as of now. See 417877eabb2335e93ddcc6ae5b1681b4426307a3.